pkg/adapter/adapter.go

```diff
@@ -41,30 +41,18 @@ import (
 
 const resourceGroup = "rabbitmqsources.sources.knative.dev"
 
-type ExchangeConfig struct {
-	Name string `envconfig:"RABBITMQ_EXCHANGE_CONFIG_NAME" required:"false"`
-}
-
-type ChannelConfig struct {
-	Parallelism int `envconfig:"RABBITMQ_CHANNEL_CONFIG_PARALLELISM" default:"1" required:"false"`
-}
-
-type QueueConfig struct {
-	Name string `envconfig:"RABBITMQ_QUEUE_CONFIG_NAME" required:"true"`
-}
-
 type adapterConfig struct {
 	adapter.EnvConfig
 
-	RabbitURL      string        `envconfig:"RABBIT_URL" required:"true"`
-	Vhost          string        `envconfig:"RABBITMQ_VHOST" required:"false"`
-	Predeclared    bool          `envconfig:"RABBITMQ_PREDECLARED" required:"false"`
-	Retry          int           `envconfig:"HTTP_SENDER_RETRY" required:"false"`
-	BackoffPolicy  string        `envconfig:"HTTP_SENDER_BACKOFF_POLICY" required:"false"`
-	BackoffDelay   time.Duration `envconfig:"HTTP_SENDER_BACKOFF_DELAY" default:"50ms" required:"false"`
-	ChannelConfig  ChannelConfig
-	ExchangeConfig ExchangeConfig
-	QueueConfig    QueueConfig
+	RabbitURL     string        `envconfig:"RABBIT_URL" required:"true"`
+	Vhost         string        `envconfig:"RABBITMQ_VHOST" required:"false"`
+	Predeclared   bool          `envconfig:"RABBITMQ_PREDECLARED" required:"false"`
+	Retry         int           `envconfig:"HTTP_SENDER_RETRY" required:"false"`
+	BackoffPolicy string        `envconfig:"HTTP_SENDER_BACKOFF_POLICY" required:"false"`
+	BackoffDelay  time.Duration `envconfig:"HTTP_SENDER_BACKOFF_DELAY" default:"50ms" required:"false"`
+	Parallelism   int           `envconfig:"RABBITMQ_CHANNEL_CONFIG_PARALLELISM" default:"1" required:"false"`
+	ExchangeName  string        `envconfig:"RABBITMQ_EXCHANGE_CONFIG_NAME" required:"false"`
```

Comment:
nit: the env vars can be simplified. No need for `CONFIG` in there. Can be `RABBITMQ_EXCHANGE_NAME` for ex